generic/ltp.py

```diff
@@ -112,18 +120,69 @@ def setUp(self):
         os.chdir(ltp_dir)
         build.make(ltp_dir, extra_args='autotools')
         if not self.ltpbin_dir:
             self.ltpbin_dir = os.path.join(self.teststmpdir, 'bin')
         if not os.path.exists(self.ltpbin_dir):
             os.mkdir(self.ltpbin_dir)
+
+        if self.two_host_configuration and "-f net" in self.args:
+            self.session = Session(self.peer_public_ip, user=self.peer_user,
+                                   password=self.peer_password)
+            if not self.session.connect():
+                self.cancel("failed connecting to peer")
+            # setting ltp direcory in peer LPAR for 2 host configuration tests
+            destination = "%s:/home" % self.peer_public_ip
+            output = self.session.copy_files(self.ltpdir, destination,
+                                             recursive=True)
+            if not output:
+                self.cancel("unable to copy the ltp into peer machine")
+            time.sleep(10)
+            cmd = "cd %s;make autotools;./configure;make;make install" % ltp_dir
+            output = self.session.cmd(cmd)
+            if not output.exit_status == 0:
+                self.cancel("Unable to compile ltp in peer machine")
+
+            # Adding Rhost name and passwd in tst_net.sh for ltp between 2 host
+            output = self.session.cmd('hostname')
+            if not output.exit_status == 0:
+                self.cancel("Unable to get the hostname of peer machine")
+            ltp_tstnet_dir = os.path.join(ltp_dir, 'testcases/lib/tst_net.sh')
+            if not os.path.exists(ltp_tstnet_dir):
+                self.log.info("File tst_net.sh does not exist")
+
+            ltp_tstnet_dir_copy = os.path.join(ltp_dir,
+                                               'testcases/lib/tst_net_copy.sh')
+            shutil.copy(ltp_tstnet_dir, ltp_tstnet_dir_copy)
+            replacements = [("export RHOST=\"$RHOST\"",
+                            "export RHOST=\"" + str(output.stdout) + "\""),
+                            ("export PASSWD=\"${PASSWD:-}\"",
+                            "export PASSWD=\"" + self.peer_password + "\"")]
+            with open(ltp_tstnet_dir_copy, 'r') as input_file, open(ltp_tstnet_dir, 'w') as output_file:
+                for line in input_file:
+                    if replacements:
+                        for old_string, new_string in replacements:
+                            line = line.replace(old_string, new_string)
+                    output_file.write(line)
+            os.remove(ltp_tstnet_dir_copy)
+
         process.system('./configure --prefix=%s' % self.ltpbin_dir)
         build.make(ltp_dir)
         build.make(ltp_dir, extra_args='install')
 
+        # necessary services to be started on the host before test run starts
+        services = ['nfs-server', 'rpcbind', 'httpd', 'vsftpd']
+        for service in services:
+            cmd = "service " + service + " restart"
```

Comment:
make use of Service Manager utils 
`from avocado.utils import service`
`Manageservice = ServiceManager()`
`Manageservice.stop("service")`

Reply:
make sure service names same for suse

Reply:
@abdhaleegit As suggested added proper service names for different distros while installation as well as restarting services.

Test run for above changes done

Fetching asset from ltp.py:LTP.test
Fetching asset from ltp.py:LTP.test
JOB ID     : 5cf2684b4929e15662dc21dc2812e84028087f32
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2023-09-24T08.52-5cf2684/job.log
 (1/1) ltp.py:LTP.test;run-runltp-net.ipv6-eb32: STARTED

 (1/1) ltp.py:LTP.test;run-runltp-net.ipv6-eb32: PASS (409.63 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2023-09-24T08.52-5cf2684/results.html
JOB TIME   : 435.09 s

Reply:
@TasmiyaNalatwad thanks